darwin/datetimepicker.m

```diff
@@ -40,3 +120,36 @@
 {
 	return finishNewDateTimePicker(NSHourMinuteSecondDatePickerElementFlag);
 }
+
+void uiDateTimePickerTime(uiDateTimePicker *d, struct tm *time)
```

Comment:
Could these methods be moved above the constructor, like in the other files?

Reply:
Will do. `libui` constructors go at the end of the file. I didn't pickup on that convention until just now.

Reply:
Will do. libui constructors go at the end of the file. I didn't pickup on that convention until just now.

Reply:
Fixed.